core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -1887,6 +1896,7 @@ public List<SubjectVersion> listVersionsForId(int id, String subject, boolean lo
       if (schema == null) {
         return null;
       }
+      logSchemaOp(toSchemaEntity(schema), "READ");
```

Comment:
Let's add `Schema schemaEntity = toSchemaEntity(schema)` before this line so we can reuse it in the rest of this method